runtime/onert/backend/cpu/ops/DepthwiseConvolutionLayer.cc

```diff
@@ -186,8 +200,24 @@ void DepthwiseConvolutionLayer::configure(
   _activation = activation;
   _output = output;
   _external_context = external_context;
+  _is_hybrid = _input->data_type() == OperandType::FLOAT32 &&
+               _kernel->data_type() == OperandType::QUANT_INT8_SYMM;
 
-  if (_input->data_type() == OperandType::QUANT_INT8_ASYMM)
+  if (_is_hybrid)
+  {
+    // ensure weight is per-channel quantized.
+    int32_t kernel_input_channel = getShape(_kernel).Dims(3);
+    // zero_points comes from flatbuffer vector. Its size is within uint32_t range.
+    size_t kernel_zerop_cnt = _kernel->data_scales().size();
+    // promote to int64_t to compare int32_t and uint32_t
+    if ((int64_t)kernel_input_channel != (int64_t)kernel_zerop_cnt)
+    {
+      throw std::runtime_error{"DConv2D hybrid supports only per-channel quantized weight."};
+    }
```

Comment:
@hseok-oh Validation check is moved to `DepthwiseConvolutionLayer.cc` from `OperationValidator.cc`.